doc/design/relocatable_pointer.md

```diff
@@ -19,100 +19,109 @@ bytes. Specifically we distinguish between management and payload segments. The
 used by RouDi and the runtime to communicate, store common structures such as ports and so on. The payload segment
 contains the actual data samples sent by senders in some application to receivers in other applications.
 
-## Relocatable pointer
+## Relocatable Pointers
 
 When pointer and pointee are located in the same shared memory segment, the lightweight (compared to a relative pointer)
 relocatable pointer can be used. That is, we have the following scenario.
 
-Pointer p points to object X of type T and both are stored in shared memory segment S.
+Pointer `p` points to object `X` of type `T` and both are stored in shared memory segment S.
 
-```
+```txt
 Shared Memory S:  p                    X
                   |____________________^
 
-App1          a1  b1                  c1 
+App1          a1  b1                  c1
 App2          a2  b2                  c2
 ```
 
-Let a1, b1, c1 and so on be the addresses of segment S, pointer p and object X in application 1 and similarly a2, b2 and
+Let a1, b1, c1 and so on be the addresses of segment S, pointer `p` and object `X` in application 1 and similarly a2, b2 and
```

Comment:
Is `p` and `X` more special than `a1`, etc. ... or to phrase it differently, doesn't this introduce inconsistencies

Reply:
I thought so as well, they are closer to language entities (pointers, values ... but so are addresses) but I will just not highlight any of them I think because it is not worth it.